src/Controls/src/Core/Entry/Entry.iOS.cs

```diff
@@ -26,5 +29,17 @@ public static void MapText(IEntryHandler handler, Entry entry)
 
 		public static void MapText(EntryHandler handler, Entry entry) =>
 			MapText((IEntryHandler)handler, entry);
+
+		private static void MapFormatting(IEntryHandler handler, IEntry entry)
+		{
+			handler.PlatformView?.UpdateMaxLength(entry);
```

Comment:
Instead of calling these directly can you propagate through the handler?

handler.UpdateValue(nameof(IEntry.MaxLength))

Reply:
Done :)

Reply:
thx!